_layouts/redirect.html

```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+---
+
+<meta http-equiv="refresh" content="0; url={{site.tumblr_archive}}{{page.permalink}}" />
+
+<h1>This page has been archived</h1>
+<a href="{{site.tumblr_archive}}{{page.permalink}}" title="redirect">Click here</a> if not redirected.
+
+<script>
+	window.location.href = '{{site.tumblr_archive}}{{page.permalink}}';
```

Comment:
The page redirects too quickly to read the message, should we add a delay?

Also, what about saying like, "The list of past events can be found {past events link}, to view the old website, click here {tumblr link}"